server/app.js

```diff
@@ -7,22 +7,27 @@ const passport = require("passport");
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
+// Auth routes
 const registerRouter = require("./routes/auth/register.route");
 const loginRouter = require("./routes/auth/login.route");
+// Poll routes
+const addPollRouter = require("./routes/polls/add.route");
+const deletePollRouter = require("./routes/polls/delete.route");
+const updatePollRouter = require("./routes/polls/update.route");
 
 const { json, urlencoded } = express;
 
 const app = express();
 const mongoose = require("mongoose"); //For database connection
 
 // Connecting to database. fail if not able to connect
-mongoose.connect("mongodb://localhost:27017", {useNewUrlParser:true});
+mongoose.connect("mongodb://localhost:27017/test", { useUnifiedTopology: true, useNewUrlParser: true });
```

Comment:
I probably should have commented this earlier, but this URL should be coming from a config file and then from eventually an environment variable